### DISCUSSION TITLE
CrowdSec "can't install on Proxmox" -- LXC specific docs

### DISCUSSION THREAD

**Discussion by cwmoriarty** [tteck/Proxmox#3360] [General] [answered]
When running on SSH shell works-as-intended: ⚠️  Can't Install on Proxmox 

Maybe the script and/or documentation should let the user know where to run that command (any LXC)

**Comment by tteck** [comment, marked as answer]
It does
> To Install CrowdSec, ⚠️ run the command below in the LXC console.

**Comment by cwmoriarty** [comment]
Ahh, missed that! Great.

Maybe the error could say something similar too?
